tests/e2e/Services/Databases/DatabasesCustomServerTest.php

```diff
@@ -3242,6 +3242,168 @@ public function testAttributeUpdateEnum(array $data)
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'data' => [
+                'string' => 'string'
+            ]
+        ]);
+
+        $this->assertEquals(200, $document['headers']['status-code']);
+        $this->assertEquals('string', $document['body']['string']);
+
+        $deleteDoc = $this->client->call(Client::METHOD_DELETE, '/databases/' . $databaseId . '/collections/' . $collectionId . '/documents/' . $newDoc['body']['$id'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]));
+
+        $this->assertEquals(204, $deleteDoc['headers']['status-code']);
+
+
+        // Test Resize Down
+        $attribute = $this->client->call(Client::METHOD_PATCH, '/databases/' . $databaseId . '/collections/' . $collectionId . '/attributes/string/' . $key, array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'size' => 10,
+            'default' => '',
+            'required' => false
+        ]);
+
+        $this->assertEquals(200, $attribute['headers']['status-code']);
+        $this->assertEquals(10, $attribute['body']['size']);
+
+        // Test create new document with new size
+        $newDoc = $this->client->call(
+            Client::METHOD_POST,
+            '/databases/' . $databaseId . '/collections/' . $collectionId . '/documents',
+            array_merge([
+                'content-type' => 'application/json',
+                'x-appwrite-project' => $this->getProject()['$id'],
+                'x-appwrite-key' => $this->getProject()['apiKey']
+            ]),
+            [
+                'documentId' => 'unique()',
+                'data' => [
+                    'string' => str_repeat('a', 10)
+                ],
+                "permissions" => ["read(\"any\")"]
+            ]
+        );
+
+        $this->assertEquals(201, $newDoc['headers']['status-code']);
+        $this->assertEquals(10, strlen($newDoc['body']['string']));
+
+        // Test update document with new size
+        $document = $this->client->call(Client::METHOD_PATCH, '/databases/' . $databaseId . '/collections/' . $collectionId . '/documents/' . $document['body']['$id'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+            'x-appwrite-key' => $this->getProject()['apiKey']
+        ]), [
+            'data' => [
+                'string' => str_repeat('a', 10)
+            ]
+        ]);
+
+        $this->assertEquals(200, $document['headers']['status-code']);
+        $this->assertEquals(10, strlen($document['body']['string']));
+    }
```

Comment:
Let's add a check that we get an exception when trying to create a document with data bigger the new size